modules/nw-udn-cr.adoc

```diff
@@ -74,8 +74,8 @@ spec:
 +
 * The `subnets` field is mandatory.
 * The type for the `subnets` field is `cidr` and `hostSubnet`:
-** `cidr` is the cluster subnet and accepts a string value.
-** `hostSubnet` specifies the nodes subnet prefix that the cluster subnet is split to.
+** `cidr` specifies the supernet for your cluster. This parameter accepts a string value.
+** `hostSubnet` defines the subnet prefix that is used to subnet the supernet into each node network.
```

Comment:
this sentence IMHO reads a bit weird, but I'm not a native english speaker.

"defines the subnet prefix that is used to subnet the supernet ..." 

Can we replace the second occurance of "subnet" with "network" maybe ? 

Do we need to elaborate on the concept of supernet  ?

Reply:
I agree. "Subnet" is used a verb and it's not really a typically verb. I removed "that is used to subnet the supernet into each node network".